frontend/components/MainPage/index.tsx

```diff
@@ -51,11 +58,11 @@ export const Main = () => {
 
   // TODO: reintroduce later,  non critical
 
-  // const hideMainOlasBalanceTopBorder = [
-  //   !backupSafeAddress, // TODO: update this condition to check backup safe relative to selectedService
-  //   activeStakingProgramId === StakingProgramId.Alpha,
-  //   isAllStakingContractDetailsRecordLoaded && !hasEnoughServiceSlots,
-  // ].some((condition) => !!condition);
+  const hideMainOlasBalanceTopBorder = [
+    // !backupSafeAddress, // TODO: update this condition to check backup safe relative to selectedService
+    activeStakingProgramId === StakingProgramId.PearlAlpha,
```

Comment:
why using this check? it seems there's nothing related to this particular contract here AlertSections
actually there seems to be a lot of the others checks we need to add, maybe we should rethink this approach with hiding border and just use some hack like negative margin